cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -135,7 +135,7 @@ pub async fn create_beta_watcher<D, A, AI, BI>(
     A::BT: Clone,
     AI: Clone,
     BI: Clone,
-    A: ActorState,
+    A: ActorState + Clone + Send,
```

Comment:
What is the source of this `Clone` bound? Are we actually cloning the ActorState somewhere down here?

Reply:
A clonable state is required for the `StateStore`. Here we only use the `update` method which doesn't need `Clone` (`get` does), but that's the problem with traits that define more than one method: the constraints on the individual methods cannot be more restrictive than those on the trait itself.